.github/workflows/verible-ci.yml

```diff
@@ -5,6 +5,7 @@ on:
   pull_request:
 
 env:
+  DOCKER_BUILDKIT: 1
```

Comment:
This might need a comment about why BUILDKIT is being used?

Reply:
There is no specific reason to do so. It's just the second generation. It provides advantages but there is no specific new feature which we are using (yet). On some platforms, buildkit is enabled by default.